README.md

```diff
@@ -62,45 +114,55 @@ yarn workspace wallet-ui storybook
 - Wallet UI dapp: http://localhost:3000/
 - Storybook: http://localhost:6006/
 
-# Dapp intergation Guide
+## Dapp Integration Guide
+
+This guide is for a dApp that would be compatible only with MetaMask. To create a dApp compatible with all wallets in the Starknet ecosystem, prefer the get-starknet library. A sample dApp can be found [here](https://github.com/PhilippeR26/Cairo1JS/tree/main/src). If you still want to directly interact with the snap follow the tutorial below:
```

Comment:
i think we dont need to refer to external dapp? as the current dapp in this repo already explain how to do Integration?

Reply:
It's just that in the current README.md we don't explain how to do multi-wallet compatible dApp on starknet. To do so dev need to use get-starknet official repo and not direct call to the snap. Calling the snap directly as described in the README.md shows how to do a dApp on starknet compatible with `metamask only`. 

The goal here is to clearly tell to devs what is the best dev strategy in terms of wallet interoperability with the starknet ecosystem. 

wdyt ?